docs/includes/metrics/prometheus-exemplar-support.adoc

```diff
@@ -37,67 +60,100 @@ Declare the following dependency in your project:
 </dependency>
 ----
 
+Also, include the Helidon integration module for a tracing implementation...for example
```

Comment:
Does the ellipsis indicate something is missing and will be added?

Reply:
It looks as if this was already fixed in an earlier revision. Now reads "Also, include the Helidon integration module for a tracing implementation (such as" and then the SE or MP `ifdef` supplies the rest of the sentence.